src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,6 +32,452 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class PublicationServiceTests
     {
+        [Fact]
+        public async Task GetPublicationsByTopic_CanViewAllPublications()
```

Comment:
Just a note that if we make any renames, we shouldn't forget to change the test method names in here as well